braindecode/preprocessing/preprocess.py

```diff
@@ -274,23 +275,53 @@ def zscore(data):
     return zscored
 
 
+def robust_scale(data):
```

Comment:
@hubertjbanville I don't get why we have this robust_scale function and changes in plot_sleep_staging.py in this PR. It's not already in master? it's required here for usleep training?

Reply:
There wasn't a robust scaling implementation yet in braindecode. It's required here as it's the only preprocessing step used in the U-Sleep paper. We could use a simple zscore but it was simple enough to implement so I added it in.

The changes to `plot_sleep_staging.py` are just because so the two sleep staging examples are more uniform.